cmd/cl-adm/cmd/create/create_peer.go

```diff
@@ -18,16 +18,13 @@ import (
 	"os"
 	"path/filepath"
 
-	"github.com/clusterlink-net/clusterlink/cmd/cl-adm/templates"
-	"github.com/clusterlink-net/clusterlink/pkg/controlplane/api"
-	dpapi "github.com/clusterlink-net/clusterlink/pkg/dataplane/api"
-
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 	"golang.org/x/net/idna"
 
 	"github.com/clusterlink-net/clusterlink/cmd/cl-adm/config"
-	"github.com/clusterlink-net/clusterlink/cmd/cl-adm/util"
+	"github.com/clusterlink-net/clusterlink/pkg/bootstrap"
```

Comment:
nit: Is `pkg/bootstrap` the right hierarchy? 
Feels like it is missing some context as to what we are bootstrapping...

Reply:
`pkg/bootstrap` contains code which generates certificates and configuration (e.g. k8s yaml) for deploying clusterlink.
I called it bootstrap as it is used only once in the lifecycle of a clusterlink site.
I'm open for other ideas :)

Reply:
@elevran How about `pkg/deployment` / `pkg/deploy` ?